### PR TITLE
New Policy (Configure Windows Arc-enabled machines to enable Windows Admin Center Extension)

### PR DESCRIPTION
**Pull request by psinnathurai**

### CODE REVIEW

File: policyDefinitions/Azure Arc/deploy-wac-extension-to-azure-arc-connected-machines/azurepolicy.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  "name": "0e4b8929-3957-46e0-bc05-0e34ce4c5f3e",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "properties": {
+    "displayName": "Configure Windows Arc-enabled machines to enable Windows Admin Center Extension",
+    "description": "This policy installs the Windows Admin Center Extension on Azure Arc Enabled Servers",
+    "metadata": {
+      "version": "1.0.1",
+      "category": "Azure Arc"
+    },
+    "assignPermissions": "false",
```
[the remark this answers is not in the document]
Comment (psinnathurai) [reply policyDefinitions/Azure Arc/deploy-wac-extension-to-azure-arc-connected-machines/azurepolicy.json:11]: You are right, I have corrected it.

[the remark this answers is not in the document]
Comment (Joshua-Donovan) [reply policyDefinitions/Azure Arc/deploy-wac-extension-to-azure-arc-connected-machines/azurepolicy.json]: looks great! Thanks for updating! We will review this for a merge as a team by the end of next week.

File: policyDefinitions/Azure Arc/deploy-wac-extension-to-azure-arc-connected-machines/azurepolicy.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
+{
+  "name": "0e4b8929-3957-46e0-bc05-0e34ce4c5f3e",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "properties": {
+    "displayName": "Configure Windows Arc-enabled machines to enable Windows Admin Center Extension",
+    "description": "This policy installs the Windows Admin Center Extension on Azure Arc Enabled Servers",
+    "metadata": {
+      "version": "1.0.1",
+      "category": "Azure Arc",
+      "assignPermissions": "false"
```
[the remark this answers is not in the document]
Comment (psinnathurai) [reply policyDefinitions/Azure Arc/deploy-wac-extension-to-azure-arc-connected-machines/azurepolicy.json]: I have adjusted. But regarding the Parameter I understood that this Parameter make sure that, if you need a Managed Identity for remediation, that the Azure Policy doesn't try to set this Permission on the Managed Identity as I as a Admin which needs to maintain Azure Policies doesn't have the Permission to give other Permission (Owner or RBAC Admin)

**Comment by psinnathurai** [comment]
Any update?

**Comment by Joshua-Donovan** [comment]
> Any Update?

Hey @psinnathurai the team will be reviewing pull requests in the next few days, sorry this one has taken so long.

**Review by Joshua-Donovan — APPROVED** [review]

**Pull request merged**